examples/simple/init_data.py

```diff
@@ -1,48 +0,0 @@
-#!/usr/bin/env python
-import json
-
-from learnosity_sdk.request import DataApi
```

Comment:
Ah, so we've lost the api-data example? It might be worth making a note of this, so we can re-add one soon. It's important as it shows the use of the other part of the SDK, via the DataApi object.

Reply:
Oh. Right. We can lodge it in the REFERENCE.md, perhaps, as a secondary Data API example?